layouts/partials/navbar.html

```diff
@@ -0,0 +1,37 @@
+{{ $cover := .HasShortcode "blocks/cover" }}
+<nav class="js-navbar-scroll navbar navbar-expand navbar-dark {{ if $cover}} td-navbar-cover {{ end }}flex-column flex-md-row td-navbar">
+        <a class="navbar-brand" href="{{ .Site.Home.RelPermalink }}">
+		<span class="navbar-logo">{{ if .Site.Params.ui.navbar_logo }}{{ with resources.Get "icons/logo.svg" }}{{ ( . | minify).Content | safeHTML }}{{ end }}{{ end }}</span><span class="text-uppercase font-weight-bold">{{ .Site.Title }}</span>
+	</a>
```

Comment:
Here it's indented with many spaces while in footer.html it's only two spaces, better to be coherent and use 2 spaces everywhere

Reply:
(same for other partials)

Reply:
Agree

Reply:
This is how I took them from the theme. Auto-formatting this Go-sprinkled html makes it really bad, sometimes breaks stuff.